shared/files/agent/montagu-docker-login

```diff
@@ -13,10 +13,20 @@ REGISTRY_URL=docker.montagu.dide.ic.ac.uk:5000
 REGISTRY_USER=vimc
 REGISTRY_PASSWORD=$(vault read -field=password /secret/registry/${REGISTRY_USER})
 
+HUB_USER=vimcrobot
+HUB_PASSWORD=$(vault read -field=password /secret/${HUB_USER)
```

Comment:
To keep the vault somewhat tidy - would you mind moving this to `/secret/vimc-robot/dockerhub` - perhaps also add a `username=vimcrobot` field to that entry?